tools/qemu_args.sh

```diff
@@ -27,6 +29,7 @@ COMMON_QEMU_ARGS="\
     -display none \
     -serial chardev:mux \
     -monitor chardev:mux \
+    -monitor unix:${SOCK_MONITOR},server,nowait \
```

Comment:
Will Asterinas in other scenarios like just executing `make run`, leave a `/run/asterinas_monitor.socket` while no one would use it ?

Reply:
An alternative is launching Asterinas by QEMU after `make build BENCHMARK=${benchmark} ENABLE_KVM=1 RELEASE_LTO=1 2>&1`. So we can add it into our QEMU commands. However, the commands are different from Linux and Asterinas, which could be ugly. Therefore, I am not fully support this approach. I think it does not matter if we use the current approach.

```sh
make run BENCHMARK=${benchmark} ENABLE_KVM=1 RELEASE_LTO=1 2>&1
qemu-system-x86_64 \
    -drive file=/root/asterinas/target/osdk/aster-nix/aster-nix-osdk-bin.iso,format=raw,index=2,media=cdrom \
    -cpu Icelake-Server,+x2apic \
    -smp 1 \
    -m 8G \
    --no-reboot \
    -nographic \
    -display none \
    -serial chardev:mux \
    -monitor chardev:mux \
    -machine q35,kernel-irqchip=split \
    -accel kvm \
    -chardev stdio,id=mux,mux=on,signal=off,logfile=qemu.log \
    -netdev user,id=net01,hostfwd=tcp::61382-:22,hostfwd=tcp::27828-:8080,hostfwd=tcp::40820-:6379,hostfwd=tcp::55508-:5201,hostfwd=tcp::44295-:31234 \
    -device isa-debug-exit,iobase=0xf4,iosize=0x04 \
    -device virtio-blk-pci,bus=pcie.0,addr=0x6,drive=x0,serial=vext2,disable-legacy=on,disable-modern=off \
    -device virtio-blk-pci,bus=pcie.0,addr=0x7,drive=x1,serial=vexfat,disable-legacy=on,disable-modern=off \
    -device virtio-keyboard-pci,disable-legacy=on,disable-modern=off \
    -device virtio-net-pci,netdev=net01,disable-legacy=on,disable-modern=off \
    -device virtio-serial-pci,disable-legacy=on,disable-modern=off \
    -device virtconsole,chardev=mux \
    -drive if=none,format=raw,id=x0,file=./test/build/ext2.img \
    -drive if=none,format=raw,id=x1,file=./test/build/exfat.img \
    -drive if=pflash,format=raw,unit=0,readonly=on,file=/usr/share/OVMF/OVMF_CODE.fd \
    -drive if=pflash,format=raw,unit=1,file=/usr/share/OVMF/OVMF_VARS.fd
```